### PR TITLE
Color all rows on comment table

### PR DESCRIPTION
**Pull request by penk85**
WPC-340: Update comment list so rows are colored without hover.

### CODE REVIEW

File: packages/editor/src/client/global.less [hunk 1]
```diff
@@ -4,29 +4,36 @@
...
-  &-hover &-row.rejected &-cell {
+  &-row.rejected:not(&-row-header),
+  &-row.rejected &-cell-group,
+  &-row.rejected &-cell {
     background: rgb(83, 85, 83);
     color: white;
     text-decoration: line-through;
   }
+
+  &-hover &-row.rejected:not(&-row-header):hover,
```
Comment (bishoymelek) [inline packages/editor/src/client/global.less:33]: What does this one do?

**Review by bishoymelek — COMMENTED** [review]
Just one point

File: packages/editor/src/client/global.less (hunk 1 above)
Comment (penk85) [reply packages/editor/src/client/global.less:33]: @bishoymelek the rejected row is colored gray with white text. On hover the background becomes light blue and the text is no longer visible so I set it black on hover.

**Review by bishoymelek — APPROVED** [review]
:tada:

**Review by HanaAyman — APPROVED** [review]

**Pull request merged**